completion_time_analysis.py

```diff
@@ -0,0 +1,237 @@
+"""
+Feature 2: Issue Completion Time Analysis (Focused)
+
+- Computes time-to-close for CLOSED issues only (no fallback).
+- ONE chart: Monthly median completion time (overall + top 3 labels by closed count).
+- Short summary: count, median, mean, P90; fastest/slowest labels if sample >= 3.
+"""
+
+from typing import List, Dict, Optional, Any
+from datetime import datetime, timezone
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from data_loader import DataLoader
+from model import Issue, State
+import config
+
+
+# ----------------------------- helpers ------------------------------------ #
+
+def _to_utc_dt(x: Any) -> Optional[datetime]:
+    if x is None:
+        return None
+    if isinstance(x, datetime):
+        return x if x.tzinfo else x.replace(tzinfo=timezone.utc)
+    ts = pd.to_datetime(str(x), utc=True, errors="coerce")
+    if pd.isna(ts):
+        return None
+    return ts.to_pydatetime()
+
+def _created_at(issue: Issue) -> Optional[datetime]:
+    """Get created date from issue."""
+    return _to_utc_dt(issue.created_date) if issue.created_date else None
+
+def _updated_at(issue: Issue) -> Optional[datetime]:
+    """Get updated date from issue."""
+    return _to_utc_dt(issue.updated_date) if issue.updated_date else None
```

Comment:
`issue.created_date` and `issue.updated_date` are already `datetime`s. And I don't think the dates need to be converted to UTC for any particular reason. So I think we don't need these.